src/test/java/com/xceptance/neodymium/testclasses/multibrowser/BrowserstackHomePageTest.java

```diff
@@ -0,0 +1,52 @@
+package com.xceptance.neodymium.testclasses.multibrowser;
+
+import static com.codeborne.selenide.CollectionCondition.sizeGreaterThan;
+import static com.codeborne.selenide.Condition.exist;
+import static com.codeborne.selenide.Condition.matchText;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$$;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.codeborne.selenide.Selenide;
+import com.xceptance.neodymium.NeodymiumRunner;
+import com.xceptance.neodymium.module.statement.browser.multibrowser.Browser;
+
+@RunWith(NeodymiumRunner.class)
+@Browser("safari_remote")
+public class BrowserstackHomePageTest
+{
+    @Test
+    public void testVisitingHomepage()
+    {
+        // Goto the home page
```

Comment:
Please add some assertions to verify that this is a Browserstack safari WebDriver as far as possible.

Reply:
Done